src/test/java/com/cdancy/bitbucket/rest/features/TasksApiLiveTest.java

```diff
@@ -49,7 +50,12 @@
     private final String taskComment = TestUtilities.randomString();
     private int commentId = -1;
     private int taskId = -1;
-    
+    private int pullRequestId = -1;
+    private int repositoryId = -1;
+
+    private static final String TASK_OPEN = "OPEN";
```

Comment:
I might just make these strings in the test - no need for a constant that's only used once.

Reply:
Agreed.

Reply:
Agreed.